README.md

```diff
@@ -26,6 +26,25 @@ Key features include:
 
 ## 🗓️ News
 
+🌟 **[2025-11-21]**
+Recent improvements include:
```

Comment:
[EASI v0.1.1](https://github.com/EvolvingLMMs-Lab/EASI/releases/tag/0.1.1) is released. Major updates include:

Reply:
It is ok to give the link of v0.1.1. We will tag it right after the PR.